app/api/events.py

```diff
@@ -577,11 +578,21 @@ def before_update_object(self, event, data, view_kwargs):
             else:
                 event.deleted_at = data.get('deleted_at')
 
+        deleted_tickets = 0
+        for ticket in event.tickets:
+            if ticket.deleted_at:
+                deleted_tickets += 1
```

Comment:
?

Reply:
If the event has more than 1 ticket associated with it, we want to make sure that every one of them is deleted. The event has tickets object even when all of them are soft deleted.